specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/cosmos-db.json

```diff
@@ -5274,6 +5274,359 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/cassandraKeyspaces/{keyspaceName}/views": {
```

Comment:
>views [](http://example.com/codeflow?start=168&length=5)

shouldn't we use the full name materializedViews, unless "views" is a popular industry word for MVs.

Reply:
Native Cassandra stores the materialized views as system_schema.views, so considered using views in resource path